wiki/cpp-tutorial/language-basics.md

```diff
@@ -81,86 +81,90 @@ int foo = 0;   // declaration statement
 foo = foo + 2; // expression statement
 ```
 
-The above snippet creates `foo` with an initial value of `1`, then evaluates `foo + 2` before assigning the value back
-to the variable `foo`.
+In above snippet, the variable `foo` is initialized with a value of `0`. The expression `foo + 2` is then 
+evaluated, and then the result is assigned back to `foo`.
 
 ## Identifiers
 
-In C++, _identifiers_ give our declarations a name. In the example above, `foo` is an identifier. Identifiers can
+In C++, _identifiers_ give our declarations a name. In the above example, `foo` is an identifier. Identifiers can
 contain alphabetic characters, `_`, digits, and certain Unicode characters, but they cannot start with a digit.
 
 ::: info For Advanced Developers
 
-While Unicode characters can be used in C++ identifiers, the degree of support for this has changed throughout the
-years. Many C++ developers prefer to use only ASCII characters in their code, which is most portable and easiest to type
-on all keyboards. For example, most developers would call a variable `alpha` instead of `α`.
+Although Unicode characters can be used in C++ identifiers, the degree of support for them has varied over the
+years. Many C++ developers prefer to stick to ASCII characters in their code, which is more portable and easier to type
+on most keyboards. For example, most developers would name a variable `alpha` instead of `α`.
 
 :::
 
 ## Keywords
 
-C++ reserves a set of words in the language for its own use. Developers may use these keywords, but they cannot be used
-as identifiers. You can find a list of all keywords on [cppreference](https://en.cppreference.com/w/cpp/keyword).
+C++ reserves a set of words in the language for its own use, known as _keywords_. These keywords may be used 
+by developers, but they cannot be used as identifiers(such as variable name). You can find a complete list of C++
+keywords on [cppreference](https://en.cppreference.com/w/cpp/keyword).
 
 ```cpp
-int my_integer_value = 0; // int is a keyword
-float pi = 3.14f;         // float is a keyword
+int my_integer_value = 0; // "int" is a keyword
+float pi = 3.14f;         // "float" is a keyword
 
 int int = 0;              // error: cannot use "int" as an identifier (variable name)
 ```
 
 ::: info For Advanced Developers
 
-In addition to these "true keywords", C++ also has "contextual keywords", formally called _identifiers with special
-meaning_, such as `override`. We will cover these in later chapters.
+In addition to these _true keywords_, C++ also has _contextual keywords_, formally called _identifiers with special
```

Comment:
I don't see why this should be italicized. "True keywords" and "contextual keywords" are colloquial terms, and my impression is that we use italics only for proper definitions.